### PR TITLE
add airbyte blog

### PR DESCRIPTION
**Pull request by brianzhan**

### CODE REVIEW

**Comment by brianzhan** [comment]
cc @mosabua

File: _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title:  "An Opinionated Guide to Consolidating our Data"
+author: Abhi Vaidyanatha
+excerpt_separator: <!--more-->
```
Comment (mosabua) [inline _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md:5]: Add 

canonical_url: 'https://abhi-vaidyanatha.medium.com/an-opinionated-guide-to-consolidating-your-data-b09386b2b9b5'

File: _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+love and [tooling](https://tailwindcss.com/) as other communities, so we were
+expected to make do with whatever came our way. In order to perform the wildly
+basic task of moving our data around, we were asked to tithe large sums of money
+to the closed-source ELT overlords.
+
+So where does that leave us? Thankfully things have changed, so here’s how you
+can move all your data to a central location for free (well, minus the
+infrastructure costs) while making few architectural choices.
+
+## The Tool
```
Comment (mosabua) [inline _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md:39]: Empty line after title

File: _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+For consolidating your data, use Airbyte. It’s straightforward to setup,
+requires minor configuration, and has tightly scoped responsibilities.
+
+<p align="center">
+   <img align="center" width="50%" src="https://miro.medium.com/max/640/1*zqLMo7P3o_HG7EJ2E1dbpg.png"/>
+</p>
+
+
+## The Destination
```
Comment (mosabua) [inline _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md:69]: The destination

File: _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+this, head to the top right of the screen in the AWS Management Console where it
+says your email provider and then click on **Security Credentials**. Click
+**Create New Access Key** and save that information for later.
+
+<p align="center">
+   <img align="center" width="50%" src="https://miro.medium.com/max/1202/1*mYeldXLcvi7iPBDZ1GKEug.png"/>
+</p>
+
+
+## The Deployment
```
Comment (mosabua) [inline _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md:97]: The deployment

File: _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+docker-compose up
+```
+
+Once you see the following banner, you’re good to go.
+
+<p align="center">
+   <img align="center" width="50%" src="https://miro.medium.com/max/1148/1*7Fg7Vwi5vgkg94SYRuACLQ.png"/>
+</p>
+
+## The Data Sources
```
Comment (mosabua) [inline _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md:125]: The data sources

File: _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+[columnar storage
+format](https://www.qubole.com/tech-blog/columnar-format-in-data-lakes-for-dummies/),
+which is optimized for reads. JSON, CSV, and Avro are supported, but will be
+less performant on read.
+
+<p align="center">
+   <img align="center" width="50%" src="https://miro.medium.com/max/1400/1*tVw2sbTLYDlHpKB97M7cKg.png"/>
+</p>
+
+## The Connection
```
Comment (mosabua) [inline _posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md:175]: The connection

**Review by mosabua — APPROVED** [review]
This is a repost .. and there are only minimal issues in the PR in terms of multiple empty lines or missing empty lines that do not affect output. As such I think we can merge.

**Pull request merged**